### DISCUSSION TITLE
Have trouble to use 3.0.0

### DISCUSSION THREAD

**Discussion by bonneforyou** [sebastienros/jint#1751] [Q&A]
3.0.0 first downloader

I am using .Net standard 2.1 in Unity, so I choose 2.1 version for jint and Esprima, and I found engine.Execute has error? Can you have an article like this one "https://pleasenophp.github.io/posts/using-real-javascript-with-unity.html" to show how to interact with new version?

btw, even I switch to 2.11.58, I also got this error

ApplicationException: Error: Line 1: Unexpected token ) in app
001 (()=>{"use strict";log("Hello from JS ES6 file!"),log("This is function 1 from my module"),log("This is function 2 from my module")})();

**Comment by lahma** [comment]
Maybe you could ask the article author for an updated version?
